src/webhook/test_namespace_validation.py

```diff
@@ -84,33 +90,55 @@ def runtest(self, namespace, groups, expect):
                 failmsg)
 
     def test_deny(self):
-        # In order to get DENYs, we must have *both* a privileged namespace
-        # *and* the 'dedicated-admins' group.
-        for ns in PRIVILEGED_NAMESPACES:
-            for gl in GROUP_LISTS:
-                # Always include dedicated-admins
-                groups = gl + ('dedicated-admins',)
-                self.runtest(ns, groups, False)
+        # layered-cs-sre-admins should not have access to any privledged namespaces except ones starting with 'redhat'
+        # dedicated-admins should not have access to any privledged namespaces
+        for ns in self.PRIVILEGED_NAMESPACES:
+            for user in self.NON_PRIVILEGED_USERS:
+                groups = ('layered-cs-sre-admins',)
+                self.runtest(ns, groups, user, False)
+                groups = ('dedicated-admins',)
+                self.runtest(ns, groups, user, False)
+                groups = ('random-test-group',)
+                self.runtest(ns, groups, user, False)
```

Comment:
need to run the same tests again on REDHAT_NAMESPACES for dedicated-admins or random-test-group users
```suggestion
                self.runtest(ns, groups, user, False)
        for ns in self.REDHAT_NAMESPACES:
            for user in self.NON_PRIVILEGED_USERS:
                groups = ('dedicated-admins',)
                self.runtest(ns, groups, user, False)
                groups = ('random-test-group',)
                self.runtest(ns, groups, user, False)
```